docs/opentelemetry/otel-fields-mapping.asciidoc

```diff
@@ -0,0 +1,2568 @@
+////
+This file is automatically generated. Don't edit it manually!
+////
+The following table gives an overview of mappings between individual ECS fields (in ECS version `9.0.0-dev`)
+and corresponding OTel semantic convention attributes (in SemConv version `1.29.0`).
+
+[%header]
+|====
+| ECS Field | Relation | OTel Semantic Conventions Attribute | Stability
+
+[[otel-mapping-namespace-base]]
+4+e| *Base Fields*
+
+.3+|
+[[otel-mapping-for-timestamp]]
+<<field-timestamp, @timestamp>> 
+
+
+
+^| image:https://img.shields.io/badge/OTLP-ffdcb2?style=flat[relation,title=otlp,link=ecs-opentelemetry.html#ecs-opentelemetry-relation]
+
+<| https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-proto+%22\+time_unix_nano+%22&type=code[time_unix_nano] +
+
+
+^|image:https://img.shields.io/badge/✔-93c93e?style=flat[stable,title=stable]
+
+// ===============================================================
+
+
+^| image:https://img.shields.io/badge/OTLP-ffdcb2?style=flat[relation,title=otlp,link=ecs-opentelemetry.html#ecs-opentelemetry-relation]
+
+<| https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-proto+%22\+observed_time_unix_nano+%22&type=code[observed_time_unix_nano] +
+
+
+^|image:https://img.shields.io/badge/✔-93c93e?style=flat[stable,title=stable]
+
+// ===============================================================
+
+
+^| image:https://img.shields.io/badge/OTLP-ffdcb2?style=flat[relation,title=otlp,link=ecs-opentelemetry.html#ecs-opentelemetry-relation]
+
+<| https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-proto+%22\+start_time_unix_nano+%22&type=code[start_time_unix_nano] +
+
+Note: On logs, events and metrics, `time_unix_nano` defines the time of the event. For logs, `observed_time_unix_nano` defines the time when the event was observed by the collection system. On spans, `start_time_unix_nano` represents the start time of the span.
+
+
+
+^|image:https://img.shields.io/badge/✔-93c93e?style=flat[stable,title=stable]
+
+// ===============================================================
+.2+|
+[[otel-mapping-for-labels]]
+<<field-labels, labels>> 
+
+
+
+^| image:https://img.shields.io/badge/related-efc20d?style=flat[relation,title=related,link=ecs-opentelemetry.html#ecs-opentelemetry-relation]
+
+<| https://opentelemetry.io/docs/specs/semconv/attributes-registry/k8s/#k8s-pod-label[k8s.pod.label] +
```

Comment:
Is there any way to link to a specific semconv version in the otel docs? I couldn't find a way. But if there is, it would be nice to link to the same `SEMCONV_VERSION` this is generated from.